src/ex4/server/services/item_manager.js

```diff
@@ -1 +1,89 @@
-// The ItemManager should go here. Remember that you have to export it.
\ No newline at end of file
+const fs = require('fs').promises;
+const tasksFile = 'tasks_file.json';
+const pokemonApi = require('../clients/pokemon_client');
+const pokemonClient = new pokemonApi();
+const cache = new Map();
+
+async function addTasksOrPokemons(tasks) {
+  let data = await readTasksFile();
```

Comment:
rename to something that is more meaningful than `data`, for example, you can rename it to `tasks`, and rename the argument of the function from `tasks` to `newTasks`